src/main/java/com/github/romankh3/image/comparison/ImageComparison.java

```diff
@@ -327,6 +336,19 @@ private void joinToRegion(int row, int col) {
         }
     }
 
+    /**
+     * Returns the list of rectangles that would be drawn as a diff image.
+     * If you submit two images that are the same barring the parts you want to mask you get a list of rectangles that can be used as said mask
+     *
+     * @return List of {@link Rectangle}
+     */
+    public List<Rectangle> createMask() {
```

Comment:
as for me, it's an open issue of how to populate these `List<Rectangle>` and how can we use it.